File: PHP/areas/rectangle.php
<?php
    class Rectangle extends FigureGeometric {
        private float $side;
        private float $calculate;

        public function __construct(float $side){
            parent::__construct("Rectangle", "Rectangulo");
            $this->side = $side;
        }
        
            
        public function getSide(): float{
            return $this->side;
        }

        public function calculate(): float {
            $this->calculate = $this->getSide() * $this->getSide();
            return $this->calculate;
        }
    }
?>

File: Java/figureGeometric/Rectangle.java
public class Rectangle extends FigureGeometric{
    private final double side;
    private double calculate;
 
    public Rectangle(double side){
        super("Rectangle", "Rectangulo");
        this.side=side;
    }

    public double getSide(){
        return this.side;
    }
    @Override
    public double calculateArea(){
        this.calculate=this.getSide() * this.getSide();
        return this.calculate;
    }
}
